Feature/customersreview

ApolinarySzumski opened the pull request.

DominikaJazwicka committed:
customers-review

ApolinarySzumski committed:
feat: customers-review section

Została opracowana mobilna wersja ww sekcji

barteknr1 commented on src/sass/components/_customersreview.scss:
```diff
@@ -0,0 +1,64 @@
+.customers-img {
+  padding: 60px 97px 32px 98px;
```
Wycentruj obrazek w poziomie za pomocą np flexa, a na dole margin odpychający recenzję. Na różnych szerokościach obrazek ucieka do lewej.

ApolinarySzumski replied:
Też, o tym myślałem, ale czemu moim zdaniem te rozwiązanie ma sens, nie wiem czy przez to nie będę musiał ubrać <img> w jakiegoś diva

barteknr1 reviewed (commented):
Jeśli chodzi o z-index to lepiej iść w górę niż w dół

barteknr1 commented on src/sass/components/_customersreview.scss:
```diff
@@ -0,0 +1,64 @@
+      rgba(0, 0, 0, 0) 30px
+    );
+  }
+  &__icon {
```
Cudzysłów trochę za mało zachodzi za napisy, dałoby radę go przesunąć trochę? :)

ApolinarySzumski replied:
dać się da na pewno :D, ale mówiłem Ci nawet wtyczkę po to specjalnie pobrałem :P

barteknr1 commented on src/sass/main.css:
```diff
@@ -0,0 +1,334 @@
+  padding-left: 21px;
+}
+.mobile-nav__item:last-child {
+  border-bottom: 1px solid #e18298;
+}
+
+.mobile-nav__link {
+  color: var(--color-white);
+  font-weight: 700;
+  font-size: var(--font-14px);
+  line-height: 1.29;
+  transition: color 250ms;
+}
+.mobile-nav__link:hover, .mobile-nav__link:focus {
+  color: var(--color-header);
+}
+
+.button {
+  height: 44px;
+  border-radius: 22px;
+  cursor: pointer;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 1.31;
+  border: none;
+}
+.button--bright {
+  color: #d41443;
+  background-color: #fff;
+}
+.button--bright:hover {
+  background-color: #ecebeb;
+}
+.button--dark {
+  color: #fff;
+  background-color: #d41342;
+}
+.button--dark:hover {
+  background-color: #c7133e;
+}/*# sourceMappingURL=main.css.map */
```
Co tu się ...?

ApolinarySzumski replied:
tego nie ruszałem

ApolinarySzumski replied:
a jeśli coś dodało w sumie nie wiem jak to nie merguj tego tylko to olej

barteknr1 reviewed (changes requested):
Moim zdaniem trzeba będzie popracować nad nazwami BEM, bo wg mnie te wyglądają tak 50/50

ApolinarySzumski committed:
fix/customers-review

Niestety z-index został jak jest bez minusów mi nie działa po prostu. Reszta ogarnięta + wyeliminowane błędy w niektórych wymiarach przez figmę od Ivana

DominikaJazwicka committed:
feature/customers

DominikaJazwicka committed:
feature/customersreview

DominikaJazwicka committed:
feature/customersreview

DominikaJazwicka committed:
feature/customersreview

DominikaJazwicka committed:
feature/customersreview

DominikaJazwicka committed:
feature/customersreview

barteknr1 committed:
Update main.scss

barteknr1 committed:
Merge branch 'feature/customersreview' of https://github.com/barteknr1/Projekt-grupowy into feature/customersreview

DominikaJazwicka committed:
Update customers-review.html
